python/lsst/ap/verify/measurements/performance.py

```diff
@@ -0,0 +1,71 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST Corporation.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+
+"""Code for measuring software performance metrics.
+
+All measurements assume the necessary information is present in a Task's metadata.
+"""
+
+from __future__ import absolute_import, division, print_function
+
+__all__ = ["measure_runtime"]
+
+import astropy.units as u
+
+import lsst.verify
+
+
```

Comment:
"runtime" to me sounds like "not compile-time", but I see your point. How about `code_profiling.py`?

(I'd prefer to avoid 'metrics' or 'measurement' in the name to avoid Department of Redundancy Department).